drivers/auxdisplay/auxdisplay_serlcd.c

```diff
@@ -0,0 +1,419 @@
+/*
+ * Copyright (c) 2023 Jan Henke <Jan.Henke@taujhe.de>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT sparkfun_serlcd
+
+#include <stdlib.h>
+#include <string.h>
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/drivers/auxdisplay.h>
+#include <zephyr/kernel.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_REGISTER(auxdisplay_serlcd, CONFIG_AUXDISPLAY_LOG_LEVEL);
+
+/*
+ * | in ASCII, used to begin a display command
+ */
+#define SERLCD_BEGIN_COMMAND 0x7C
+
+/*
+ * special command for the underlying display controller
+ */
+#define SERLCD_BEGIN_SPECIAL_COMMAND 0xFE
+
+/*
+ * delay in milliseconds after a normal command was sent
+ */
+#define SERLCD_COMMAND_DELAY 10
```

Comment:
Keep the unit in the constant name, `SERLCD_COMMAND_DELAY_MS`, `SERLCD_SPECIAL_COMMAND_DELAY_MS` etc...

Reply:
Done